concepts/decorators/about.md

```diff
@@ -1,2 +1,44 @@
-#TODO: Add about for this concept.
+# About
 
+Decorators are a feature of the Python language that can modify or register a function.
+
+## How to use them
+
+Decorators are placed just above a function in the code like so:
+
+```py
+@decorator
+def function():
+    pass
+```
+
+Some decorator functions take arguments:
+
+```py
+@decorator2(name="Bob")
+def function2():
+    pass
+```
+
+If a decorator function takes any arguments, but you don't wish to pass any, you must still call it with parentheses for the decorator to work:
+
+```py
+@decorator2()
+def function3():
+    pass
+```
+
+Decorators are just syntactic sugar. An alternative way of doing the same as the above three examples is using higher-order functions:

```

Comment:
By and large the docs split lines on sentence-ends with one sentence per line.

Reply:
Yeah, I realised that this one was merged just after committing, so will fix it with my next commit rather than making a single commit to fix it!